Modules/posixmodule.c

```diff
@@ -13696,6 +13696,12 @@ all_ins(PyObject *m)
 #ifdef CLD_CONTINUED
     if (PyModule_AddIntMacro(m, CLD_CONTINUED)) return -1;
 #endif
+#ifdef STATUS_CONTROL_C_EXIT  /* Windows */
+    if (PyModule_AddObject(m, "STATUS_CONTROL_C_EXIT",
+                           PyLong_FromUnsignedLong(STATUS_CONTROL_C_EXIT))) {
```

Comment:
Is this just added for test_signal? You may as well define it there. I don't think it's useful in general. `sys.exit` and `os._exit` don't support a 32-bit unsigned value. They should, but instead the value overflows. `sys.exit` end ups setting the exit status to -1 (0xFFFFFFFF) in this case, and `os._exit` raises `OverflowError`. Technically, `NTSTATUS` and `HRESULT` codes are signed values, but they're documented and often used as unsigned values (e.g. 0xC000013A). Plus the return code of a process is unsigned.

Reply:
Thanks!  I did this because I thought someone might find it useful... but I agree the mismatch between our exit API behaviors (signed, not cast) and process return code behaviors (unsigned) make constants in this range only usable by one API or the other.  Untangling that, if ever, can be its own issue.  I haven't noticed a strong demand for it.